cms/static/sass/elements/_tooltip.scss

```diff
@@ -0,0 +1,27 @@
+.tooltip {
```

Comment:
Why do we need this file? Isn't this Sass only used for the Pattern Library page?